### ISSUE TITLE
Atom ambiguity (plural forms) in kitchen-calculator exercise

### ISSUE THREAD

**Issue by lauramcastro** [exercism/elixir#880]
The kitchen-calculator exercise uses a series of atoms to represent baking units, namely (as quoted in the exercise description),  `:cup`, `:fluid_ounce`, `:teaspoon`, `:tablespoon`, and `:milliliter`.

However, the last section of the exercise, describing the `convert/2` function, features an example using a different atom for `:teaspoon`, using the plural `:teaspoons`. This should be fixed to avoid confusing students.

On further exploration, the test file kitchen_calculator_test.exs features also an occurrence of `:teaspoons` in the tests for the `get_volume/1` function, but since this is likely to be implemented ignoring the unit atom, it might have gone unnoticed so far due to this reason. There is also an occurrence of the plural `:tablespoons` in another test for the same `get_volume/1` function.

In summary,

1. The kitchen-calculator exercise features three uses of two plural forms for the unit atoms which are not specified in the exercise description, namely:
    - A `:teaspoons` in the example of use for the `convert/2` function. Should be corrected to `:teaspoon`.
    - A `:teaspoons` in one of the tests for the `get_volume/1` function. Should be corrected to `:teaspoon`.
    - A `:tablespoons` in one of the tests for the `get_volume/1` function. Should be corrected to `:tablespoon`.

I am happy to fork the repo and submit a PR!

**Comment by angelikatyborska** [comment]
@lauramcastro You're totally right. This needs to be corrected to be always singular. A PR from you would be much appreciated, thanks!

**Comment by jiegillet** [comment]
Fixed in #881. Thank you again for the PR.
